anthos-bm-openstack-terraform/cloud-config.yaml

```diff
@@ -1,17 +1,3 @@
-# Copyright 2021 Google LLC
```

Comment:
License header on the cloud-init file will prevent cloud-init from executing the steps. So we have to remove it for this file